spring-boot/proxy/src/main/java/eu/coatrack/proxy/metrics/MetricsTransmitter.java

```diff
@@ -20,7 +20,7 @@
  * #L%
  */
 
-import eu.coatrack.proxy.security.UrlResourcesProvider;
+import eu.coatrack.proxy.security.consumerAuthenticationProvider.apiKeyProvider.apiKeyFetcher.UrlResourcesProvider;
```

Comment:
This looks odd, like an "over-engineered" nested package structure. To be discussed.

Reply:
Done. Removed the over-engineered package approach and implemented a simpler one.